app/controllers/concerns/manager_api.rb

```diff
@@ -0,0 +1,24 @@
+module ManagerApi
+
+  def get_name(uniqname, program)
+    result = {'valid' => false, 'note' => '', 'last_name' => '', 'first_tname' => ''}
```

Comment:
typo on tname

Reply:
done.
Funny, it did not affect the workflow, because it returned the result like this:
`{"valid"=>true, "note"=>"", "last_name"=>"Smoke", 'first_tname' => '', "first_name"=>"Rick"}`